CK3toEU4/CMakeLists.txt

```diff
@@ -25,6 +25,7 @@ file(GLOB CONFIGURATION_SOURCES "${PROJECT_SOURCE_DIR}/Configuration/*.cpp")
 file(GLOB CK3WORLD_SOURCES "${PROJECT_SOURCE_DIR}/CK3World/*.cpp")
 file(GLOB CK3WORLD_CHARACTERS_SOURCES "${PROJECT_SOURCE_DIR}/CK3World/Characters/*.cpp")
 file(GLOB CK3WORLD_COATSOFARMS_SOURCES "${PROJECT_SOURCE_DIR}/CK3World/CoatsOfArms/*.cpp")
+file(GLOB CK3WORLD_GEOGRAPHY_SOURCES "${PROJECT_SOURCE_DIR}/CK3World/Geography/*.cpp")
```

Comment:
I'm told its preferred to instead do
```
set(CK3WORLD_GEOGRAPHY_SOURCES ${CK3WORLD_GEOGRAPHY_SOURCES} "${PROJECT_SOURCE_DIR}/CK3World/Geography/oneSourceFile.cpp")  
set(CK3WORLD_GEOGRAPHY_SOURCES ${CK3WORLD_GEOGRAPHY_SOURCES} "${PROJECT_SOURCE_DIR}/CK3World/Geography/twoSourceFile.cpp")
```

The difference I've noted is that it properly handles file updates and file removals without cleaning the whole thing. It doesn't matter for travis (which starts from scratch each time), but is nice if you're building on your own system.